src/OpenLoco/Windows/ScenarioSelect.cpp

```diff
@@ -191,15 +191,9 @@ namespace OpenLoco::Ui::Windows::ScenarioSelect
         auto isLoaded = ObjectManager::findIndex(scenarioInfo->currency);
         if (!isLoaded)
         {
-            // Unload current object
-            registers regs;
-            regs.ebp = 0x0011264A4; // currencyMeta
-            call(0x00471FF8, regs); // unload object
-
-            // Load required object
-            registers regs2;
-            regs2.ebp = X86Pointer(&scenarioInfo->currency);
-            call(0x00471BCE, regs2);
+            // Unload current currency object
```

Comment:
comment probably not needed now since code is self-explanatory

Reply:
No harm in leaving it imo